api/src/application/image/image-service.js

```diff
@@ -9,7 +9,7 @@ module.exports = () => ({
     const filename = key;
 
     const bucket = new s3({ params: { Bucket: s3Bucket } });
-    return bucket.upload({ Body: stream, Key: filename, ContentType: contentType })
+    return bucket.upload({ Body: stream, Key: filename, ContentType: contentType, CacheControl: 'max-age=0' })
```

Comment:
I did a bunch of s3 cache stuff a while back and I ended up using `public, must-revalidate, proxy-revalidate, max-age=0` for the cache control value of our index.html

I was expecting this to be the same, but I don't remember what all that stuff was (except for must revalidate, I think that would be necessary). I do know that the purpose of all that stuff is to allow it to use the cached image but it must make sure that it has the most recent version

Also if you don't want to get your local dev environment working you can do:

```
$ cd api
$ docker build -t oidc-test .
```

And then you can go into your docker-compose.yml setup on a project and replace the oidc image with your local `oidc-test` image.

Reply:
Thanks, that worked for testing. You were right, just setting `max-age=0` didn't do it, but adding those other params works.